react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+
+function CustomButton(props) {
```

Comment:
Try extracting this component out to its own file and look into props destructuring.